frontend/utilities/constants.ts

```diff
@@ -337,6 +337,83 @@ export const DEFAULT_POLICIES = [
       "Contact your IT administrator to ensure your Mac is receiving a profile that prevents guest access to shared folders.",
     platform: "darwin",
   },
+  {
```

Comment:
I have checked and those work fine in a local Fleet build (so does the query library in my local build of the website).

I'll leave reviewing of the wording to @noahtalerman except for the comments I made on the .yml that should obviously be reflected in the .ts.

Thanks!